engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/JobPreparationActor.scala

```diff
@@ -60,43 +40,24 @@ case class JobPreparationActor(executionData: WorkflowExecutionActorData,
   }
 
   def prepareJobExecutionActor(inputs: Map[LocallyQualifiedName, WdlValue]): Try[BackendJobPreparationSucceeded] = {
+
+    import RuntimeAttributeDefinition.{addDefaultsToAttributes, evaluateRuntimeAttributes}
+    val curriedAddDefaultsToAttributes = addDefaultsToAttributes(factory.runtimeAttributeDefinitions(initializationData), workflowDescriptor.backendDescriptor.workflowOptions) _
+
     for {
-      evaluatedRuntimeAttributes <- evaluateRuntimeAttributes(jobKey, expressionLanguageFunctions, inputs)
-      jobDescriptor = BackendJobDescriptor(workflowDescriptor.backendDescriptor, jobKey, evaluatedRuntimeAttributes, inputs)
+      unevaluatedRuntimeAttributes <- Try(jobKey.call.task.runtimeAttributes)
+      evaluatedRuntimeAttributes <- evaluateRuntimeAttributes(unevaluatedRuntimeAttributes, expressionLanguageFunctions, inputs)
+      attributesWithDefault = curriedAddDefaultsToAttributes(evaluatedRuntimeAttributes)
+      jobDescriptor = BackendJobDescriptor(workflowDescriptor.backendDescriptor, jobKey, attributesWithDefault, inputs)
```

Comment:
Good news: the fixes here appear to have fixed at least the `ScatterWorkflowSpec` test and likely more
Bad news: I had to drop an underscore at the end of line 26 of `RuntimeAttributesDefinition` to get that to compile